src/main/java/io/kokuwa/maven/helm/AbstractHelmMojo.java

```diff
@@ -141,6 +142,22 @@ public abstract class AbstractHelmMojo extends AbstractMojo {
 	@Parameter(defaultValue = "${project.version}", readonly = true)
 	private String projectVersion;
 
+	@Parameter(property = "helm.namespace")
+	private String namespace;
+
+	@Parameter(property = "helm.kubeApiServer")
+	private String kubeApiServer;
+
+	@Parameter(property = "helm.kubeAsUser")
+	private String kubeAsUser;
+
+	@Parameter(property = "helm.kubeAsGroup")
+	private String kubeAsGroup;
+
+	@Parameter(property = "helm.kubeToken")
+	@ToString.Exclude
```

Comment:
This is not necessary because this mojo does not have a `toString` method nor a `@ToString` annotation.